glide-core/redis-rs/redis/tests/auth.rs

```diff
@@ -0,0 +1,310 @@
+mod support;
```

Comment:
Hm, redis-rs has no licence headers. Should we add them?

Reply:
It's a good question, probably too legal, but I want to say no, it's our new code, we should do as we want.